src/test/cls_rbd/test_cls_rbd.cc

```diff
@@ -723,6 +723,10 @@ TEST_F(TestClsRbd, snapshot_limits)
 
   ASSERT_EQ(0, create_image(&ioctx, oid, 0, 22, RBD_FEATURE_LAYERING, oid, -1));
 
+  // if snapshot doesn't set limit, the limit is UINT64_MAX
+  ASSERT_EQ(0, snapshot_get_limit(&ioctx, oid, &limit));
+  ASSERT_EQ(limit, UINT64_MAX);
```

Comment:
Expectation should be listed first:

```suggestion
  ASSERT_EQ(UINT64_MAX, limit);
```

Reply:
Thanks. 
I will change later.